packages/components/src/index.ts

```diff
@@ -538,6 +579,11 @@ export {
     PermissionTypes,
     Persistence,
     SCHEMAS,
+    CBMB,
+    ASSAY_TABLES,
+    EXP_TABLES,
+    DATA_CLASSES,
+    SAMPLE_SETS,
```

Comment:
Doesn't the exported `SCHEMAS` const already make these other const available? For example, using `SCHEMAS.CBMB` or `SCHEMAS.SAMPLESETS`.

Reply:
Thanks for catching that Cory, I missed that these were all part of SCHEMAS.  Fixed